packages/forms/src/form_builder.ts

```diff
@@ -23,6 +23,18 @@ function isAbstractControlOptions(options: AbstractControlOptions|{[key: string]
        (options as AbstractControlOptions).updateOn !== undefined);
 }
 
+/**
+ * The union of all validator types that can be accepted by a ControlConfig.
+ */
+type validatorConfig = ValidatorFn|AsyncValidatorFn|ValidatorFn[]|AsyncValidatorFn[];
```

Comment:
nit: uppercase type name:
```suggestion
type ValidatorConfig = ValidatorFn|AsyncValidatorFn|ValidatorFn[]|AsyncValidatorFn[];
```